include/wx/gtk/app.h

```diff
@@ -58,6 +58,12 @@ class WXDLLIMPEXP_CORE wxApp: public wxAppBase
     // various bugs arising due to this.
     static bool GTKIsUsingGlobalMenu();
 
+    // Provide the ability to suppress GTK output. By default, all output
+    // will be suppressed, but the user can pass in a mask specifing the
+    // types of messages to suppress. Flags are defined by glib with the
+    // GLogLevelFlags enum.
+    static void GTKSuppressDiagnostics(long flags = -1);
```

Comment:
`int` would be fine for `flags`, `GLogLevelFlags` only has 9 elements.

Reply:
Changed, thanks.

Reply:
The user_data pointer is used as an integer and the flags is copied into it. My compiler gave an error if i tried to assign an integer less than a long into the user_data pointer,

Reply:
This should be fine with `wxUIntPtr` however.